### PR TITLE
Fix serialized format of ReadableSize

### LINKED ISSUE #12518: [Dynamic Region] region size config is not inconsistent between tikv config file and pd config show
### What version of TiKV are you using?
/ # ./pd-server -V
Release Version: v6.1.0-nightly
Edition: Community
Git Commit Hash: https://github.com/tikv/pd/commit/656f50d4578c50e8aa658459cc866a68c12ba6c2
Git Branch: heads/refs/tags/v6.1.0-nightly
UTC Build Time: 2022-05-13 10:56:22

/ # ./tikv-server -V
TiKV
Release Version: 6.1.0-alpha
Edition: Community
Git Commit Hash: 744d688512744c7a6898d96c4f247c1dc26680b2
Git Commit Branch: heads/refs/tags/v6.1.0-nightly
UTC Build Time: 2022-05-14 10:57:47
Rust Version: rustc 1.60.0-nightly (1e12aef3f 2022-02-13)
Enable Features: jemalloc mem-profiling portable sse test-engine-kv-rocksdb test-engine-raft-raft-engine cloud-aws cloud-gcp cloud-azure
Profile: dist_release

### What operating system and CPU are you using?
8core、32GB

### Steps to reproduce
modify tikv config file

### What did you expect?
region size config is inconsistent between tikv config file and pd config show

### What did happened?
image
image
image
middle_img_v2_4f30db8b-82c2-4cde-9022-cf7e36fa74ag

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <hustmh@gmail.com>

### What is changed and how it works?

Issue Number: Close #12518

What's Changed:

```commit-message
Fix serialized format of ReadableSize
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
Fix serialized format of ReadableSize
```

### CODE REVIEW

**Review by Connor1996 — COMMENTED** [review]
What's the difference

File: components/tikv_util/src/config.rs [hunk 1]
```diff
@@ -147,7 +147,7 @@ impl Serialize for ReadableSize {
         } else if size % KIB as u64 == 0 {
             write!(buffer, "{}KiB", size / KIB).unwrap();
         } else {
-            return serializer.serialize_u64(size);
+            write!(buffer, "{}", size).unwrap();
```
[the remark this answers is not in the document]
Comment (5kbpers) [reply components/tikv_util/src/config.rs:150]: It's intended to keep consistency with other units which are serialized as a string. Will add a unit.

**Comment by 5kbpers** [comment]
> What's the difference

It was serialized as a number, not a string, which is inconsistent with others.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by gengliqi — APPROVED** [review]

**Comment by BusyJay** [comment]
cases fail.

**Pull request merged**